src/mimemail.erl

```diff
@@ -1065,11 +1066,16 @@ encode_extra_utf_bytes(Bytes, [C|T], AccOut) when C >= 128 andalso C =< 191 ->
 %% be located in "foo.bar._domainkey.example.com" (see RFC-6376 #3.6.2.1).
 %% `t' - signature timestamp: 'now' or UTC {Date, Time}
 %% `x' - signatue expiration time: UTC {Date, Time}
+%% `a` - key algorithm:
 %% `private_key' - private key, to sign emails. May be of 2 types: encrypted and
 %% plain in PEM format:
+%% RSA
 %% `{pem_plain, KeyBinary}' - generated by <code>openssl genrsa -out out-file.pem 1024</code>
 %% `{pem_encrypted, KeyBinary, Password}' - generated by, eg
 %%  <code>openssl genrsa -des3 -out out-file.pem 1024</code>
+%% Ed25519
+%% `{pem_plain, KeyBinary}' - generated by <code>openssl genpkey -algorithm ed25519 -out out-file.pem</code>
+%% TODO: Key with password
```

Comment:
As far as I remember, the way I generated keys with passwords is smth like:
```
openssl genrsa -des3 -out deploy/keys/dkim-${DOMAIN}-private.pem 1024
```
so, `-des3` key I suppose.